MULLVAD_BUILD.md

```diff
@@ -20,14 +20,14 @@ Configure signing in Microsoft partner portal:
 
 1. Clone/pull updated driver code on trusted build machine.
 1. Launch `Developer Command Prompt for VS 2019`.
-1. Run `build.bat <certificate-sha1-thumbprint>` to build and sign the driver.
+1. Run `build.bat <certificate-sha1-thumbprint>` `<path-to-CA-root-certificate-file>` to build and sign the driver.
```

Comment:
Nit: Use a single pair of backticks: `build.bat <certificate-sha1-thumbprint> <path-to-CA-root-certificate-file>`

Reply:
Oops! My mistake.